tests/foreman/ui/test_hostcollection.py

```diff
@@ -626,3 +627,24 @@ def test_positive_remove_package_group(self):
             for package in FAKE_0_CUSTOM_PACKAGE_GROUP:
                 self._validate_package_installed(
                     self.hosts, package, expected_installed=False)
+
+    @tier3
+    def test_positive_install_errata(self):
```

Comment:
Is only one host present in hostcollection?
Anyway, do we have actual verification that errata was installed for all hosts in host collection?